tests/test_gaap.py

```diff
@@ -327,10 +327,11 @@ def testFlags(self, sigmas=[2.5, 3.0, 4.0], scalingFactors=[1.15, 1.25, 1.4, 100
         record = catalog[0]
         algorithm.measure(record, exposure)
         seeing = exposure.getPsf().getSigma()
+        pixelScale = exposure.getWcs().getPixelScale().asArcseconds()
         # Measurement must fail (i.e., flag_bigPsf and flag must be set) if
         # sigma < scalingFactor * seeing
         # Ensure that there is at least one combination of parameters that fail
-        if not(min(gaapConfig.sigmas) < seeing*max(gaapConfig.scalingFactors)):
+        if not(min(gaapConfig.sigmas)*pixelScale < seeing*max(gaapConfig.scalingFactors)):
```

Comment:
Should this be `/pixelScale` ?  

To me, the units currently appear to be [arcsec] * [arcsec/pixel] < [pixel]

Reply:
You're absolutely right.